config/helm/webhook-certificate.yaml

```diff
@@ -1,14 +0,0 @@
----
```

Comment:
What was this doing? Adding the cert-manager annotation?

Reply:
There's actually no change here. I had to split the files into `webhook-{validating,mutating}*.yaml` because some linter or tool required one manifest per file.